modules/vshn-lbaas-hieradata/templates/hieradata.yaml.tmpl

```diff
@@ -34,3 +34,7 @@ profile_openshift4_gateway::backends:
 %{ for be in backends["router"] ~}
     - ${be}
 %{ endfor ~}
+%{ if team != "" ~}
```

Comment:
If we make the variables mandatory, I'd still keep this check, as I don't think we want to require the variables to be non-empty.